core/src/main/scala/cats/mtl/instances/all.scala

```diff
@@ -9,6 +9,5 @@ trait AllInstances extends EitherTInstances
   with RaiseInstances with ReaderTInstances
   with LocalInstances with StateInstances
   with StateTInstances with WriterTInstances
-  with EmptyInstances with ReaderWriterStateTInstances
```

Comment:
Oh actually it's not odd, you deleted them here :smile:

Reply:
Cool, fixed now!